### PR TITLE
Fix IDs in MTA files

### PR DESCRIPTION
**Pull request by RichardHoch**
Partial resolution of https://issues.redhat.com/browse/WINDUP-3014: ensuring that module IDs are compliant.

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick: Please review.

File: docs/topics/web-report-app-details.adoc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide/master.adoc
-[id='report_app_details_{context}']
+
+[id='report-app-details_{context}']
```
Comment (apinnick) [inline docs/topics/web-report-app-details.adoc:4]: change to `web-report-app-details_{context}'

File: docs/topics/method-design.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 // * docs/getting-started-guide/master.adoc
-[id='design_phase_{context}']
+[id='method-design_{context}']
```
Comment (apinnick) [inline docs/topics/method-design.adoc:3]: Global change: There should be a blank line between the comment line and the id line:

```
// Module included in the following assemblies:
// * docs/getting-started-guide/master.adoc

[id='method-design_{context}']
```

Also, there should be a `//` line between the `/ Module included...` line and the `// * docs/...` line:

```
// Module included in the following assemblies:
//
// * docs/getting-started-guide/master.adoc
```


You might be able to do it with a global search/replace if you are careful and can use regex.

**Comment by RichardHoch** [comment]
@apinnick; Please review.

**Pull request merged**